frontend/admin-ui/jest.config.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
   testMatch: [
     '<rootDir>/test/jest/__tests__/**/*.spec.js',
     '<rootDir>/test/jest/__tests__/**/*.test.js',
+    '<rootDir>/test/snapshots/**/*.spec.js',
```

Comment:
snapshot testing will be run out of the box; the spec needs to be added to jest configuration